solutions/usaco-1084.mdx

```diff
@@ -17,30 +17,34 @@ Note: The video solution might not be the same as other solutions. Code in C++.
 
 ```java
 import java.util.*;
-import java.io.*;
-public class Main {
+
+public class OddPhotos {
 	public static void main(String[] args) {
 	  	Scanner sc = new Scanner(System.in);
```

Comment:
why still scanner

Reply:
matter of preference
just like scanf or cin

Reply:
@SansPapyrus683 from code conventions lol:
![image](https://user-images.githubusercontent.com/79723780/170902960-705ae46c-385d-4e23-b332-20e632bad66a.png)

Reply:
fair enough